Scenario: should allow to save account locally is broken

### Expected behaviour
`npm tun e2e-test` should end with success
### Actual behaviour
`npm tun e2e-test` ends with
```
Failures:

1) Scenario: should allow to save account locally, after page reload it should require passphrase to do the first transaction, and remember the passphrase for next transactions - features/accountManagement.feature:2
   Step: And I should see alert dialog with title "Success" and text "Your transaction of 2 LSK to 537318935439898807L was accepted and will be processed in a few seconds." - features/accountManagement.feature:23
   Step Definition: features/step_definitions/generic.step.js:92
   Message:
     AssertionError: inside element ".modal-dialog .alert-dialog-message": expected 'Request is made on the wrong network.' to equal 'Your transaction of 2 LSK to 537318935439898807L was accepted and will be processed in a few seconds.'
         + expected - actual

         -Request is made on the wrong network.
         +Your transaction of 2 LSK to 537318935439898807L was accepted and will be processed in a few seconds.
```

### Steps to reproduce
Run `npm tun e2e-test`

--- a/src/components/login/login.js
+++ b/src/components/login/login.js
@@ -129,7 +129,7 @@ class Login extends React.Component {
     const { savedAccounts } = this.props;
     if (savedAccounts && savedAccounts.length > 0 && !this.props.account.afterLogout) {
       this.account = savedAccounts[0];
-      const network = Object.assign({}, getNetwork(this.state.network));
+      const network = Object.assign({}, getNetwork(this.account.network));
       if (this.account.network === networks.customNode.code) {
         network.address = this.account.address;
       }

